src/book/01-team/02-joining/01-how-we-work.mdx

```diff
@@ -24,7 +24,7 @@ We use GitHub for development and code collaboration. All NUbots code can be fou
 
 The main repositories are:
 
-- [NUbots](https://github.com/NUbots/NUbots) - monorepo with the C++ codebase that runs on the robots, and NUsight, the web-based debugging environment we used to understand and improve our systems.
+- [NUbots](https://github.com/NUbots/NUbots) - C++ codebase that runs on the robots and NUsight, the web-based debugging environment we used to understand and improve our systems.
```

Comment:
Something here about our ability to run the codebase locally would be nice. Possibly:
```suggestion
- [NUbots](https://github.com/NUbots/NUbots) - C++ codebase that runs on the robots, or locally via simulation and NUsight, the web-based debugging environment we used to understand and improve our systems.
```